test/auth/helpers/uport-signer-test.js

```diff
@@ -0,0 +1,108 @@
+import { describe, it } from 'mocha'
+import { assert } from 'chai'
+import sinon from 'sinon'
+import sinonStubPromise from 'sinon-stub-promise'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import uPortSigner, { __RewireAPI__ } from 'modules/auth/helpers/uport-signer'
+import { Connect } from 'uport-connect'
+
+import { MODAL_UPORT } from 'modules/modal/constants/modal-types'
+
+sinonStubPromise(sinon)
+
+describe('modules/auth/helpers/uport-signer.js', () => {
+  const middlewares = [thunk]
+  const mockStore = configureMockStore(middlewares)
+  const store = mockStore({})
+
+  const stubbedUpdateModal = sinon.stub().returnsArg(0)
+  __RewireAPI__.__Rewire__('updateModal', stubbedUpdateModal)
+
+  const stubbedCloseModal = sinon.stub().returns({
+    type: 'close-stub',
+  })
+  __RewireAPI__.__Rewire__('closeModal', stubbedCloseModal)
+
+  const test = t => it(t.description, done => t.assertions(store, done))
+
+  after(() => {
+    __RewireAPI__.__ResetDependency__('updateModal')
+    __RewireAPI__.__ResetDependency__('closeModal')
+  })
+
+  afterEach(() => {
+    sinon.restore(Connect.prototype.sendTransaction)
+    store.clearActions()
+  })
+
+  test({
+    description: 'should call the expected actions from the `sendTransaction` callback',
+    assertions: (store, done) => {
+      const stubbedSendTransaction = sinon.stub(Connect.prototype, 'sendTransaction').returnsPromise()
```

Comment:
This line is in all 3 tests. Can it move to the test function above?

Reply:
Yeah, I’ll move that to a `beforeEach`.